src/components/ViewportButton/index.tsx

```diff
@@ -9,7 +9,7 @@ interface ViewportButtonProps extends ButtonProps {
     tooltipText?: string;
     tooltipPlacement?: TooltipProps["placement"];
     tooltipWhenDisabled?: boolean;
-    icon?: ReactNode | string; // When using an icomoon icon, pass the icon name as defined in selectors in src/styles.css
+    icon?: ReactNode | string; // When using an icomoon icon, pass the right IconGlyphs member as defined in src/constants/interfaces
```

Comment:
Should this comment also mention the wrapper function that adds the icomoon class names?